freshmaker/handlers/koji/rebuild_images_on_parent_image_build.py

```diff
@@ -176,7 +176,7 @@ def _verify_advisory_rpms_in_container_build(self, errata_id, container_build_id
         # and use set of the nvrs as a value.
         advisory_rpms_by_name = {}
         e = Errata()
-        build_nvrs = e.get_builds(errata_id)
+        build_nvrs = e.get_srpm_nvrs(errata_id)
```

Comment:
Please also comparing with binary rpms instead of srpms.

Reply:
Done